ui/apps/platform/src/Containers/VulnMgmt/widgets/CvesByCvssScore.js

```diff
@@ -107,45 +109,39 @@ const CvesByCvssScore = ({ entityContext, parentContext }) => {
 
     function getChildren(vulns, severity) {
         return vulns
-            .filter(
-                (vuln) =>
-                    vuln.severity === `${severity.toUpperCase()}${vulnerabilitySeveritySuffix}`
-            )
+            .filter((vuln) => vuln.severity === severity)
             .map(({ cve, cvss, summary }) => {
-                const severityString = `${severity.toUpperCase()}${vulnerabilitySeveritySuffix}`;
                 return {
-                    severity,
+                    // severity,
```

Comment:
Did you mean to leave this commented, or leave the comment in?

Reply:
Good question. Yes, I will clarify in the comment.

The prop seems unused (because unexpected in the `Sunburst` component) but I left as a comment, just in case.

Reply:
Done.